modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/PredefinedValuesTable.tsx

```diff
@@ -214,7 +215,7 @@ export default function PredefinedValuesTable({
 		values.parameters,
 	]);
 
-	useEffect(() => {
+	const getSelectedFields = useMemo(() => {
```

Comment:
This returns a function that returns a function. It looks like a use case for `useCallback`. Check [React docs](https://react.dev/reference/react/useMemo#memoizing-a-function).

This would look like as something like this:
```
	const getSelectedFields = useCallback(() => {
		const objectFields: ObjectField[] = [];

		predefinedValues?.forEach(({name}) => {
			if (objectFieldsMap.has(name)) {
				const field = objectFieldsMap.get(name);
				objectFields.push(field as ObjectField);
			}
		});

		return objectFields;
	}, [predefinedValues, objectFieldsMap]);
```

Reply:
Oh great! I will take a look at that, thanks Carlos!